modules/microshift-olm-deploy-ops-spec-ns.adoc

```diff
@@ -0,0 +1,247 @@
+    securityContextConfig: restricted <3>
+  updateStrategy:
+    registryPoll:
+      interval: 60m
+----
+<1> The global namespace. Setting the `metadata.namespace` to `openshift-marketplace` enables the catalog to run in all namespaces. Subscriptions CRs in any namespace can reference catalogs created in the `openshift-marketplace` namespace.
+<2> Community Operators are not installed by default with OLM for {miroshift-short}. Listed here for example only.
+<3> The value of `securityContextConfig` must be set to `restricted` for {microshift-short}.
+
+. Apply the `CatalogSource` configuration by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f _<my-catalog-source.yaml>_ <1>
+----
+<1> Replace _<my-catalog-source.yaml>_ with your catalog source configuration file name.
+
+. To verify that the catalog source is applied, check for `READY` state by using the following command:
++
+[source,terminal]
+----
+$ oc describe catalogsources.operators.coreos.com -n openshift-operators operatorhubio-catalog
+----
++
+.Example output
+[source,terminal]
+----
+Name:         operatorhubio-catalog
+Namespace:    openshift-operators
+Labels:       <none>
+Annotations:  <none>
+API Version:  operators.coreos.com/v1alpha1
+Kind:         CatalogSource
+Metadata:
+  Creation Timestamp:  2024-01-30T12:05:14Z
+  Generation:          1
+  Resource Version:    59582
+  UID:                 1260f123-1189-4b68-8153-841a56ee7a5d
+Spec:
+  Display Name:  Community Operators
+  Grpc Pod Config:
+    Security Context Config:  restricted
+  Image:                      quay.io/operatorhubio/catalog:latest
+  Publisher:                  OperatorHub.io
+  Source Type:                grpc
+  Update Strategy:
+    Registry Poll:
+      Interval:  60m
+Status:
+  Connection State:
+    Address:                   operatorhubio-catalog.openshift-operators.svc:50051
+    Last Connect:              2024-01-30T12:05:42Z
+    Last Observed State:       READY
+  Latest Image Registry Poll:  2024-01-30T13:05:14Z
+  Registry Service:
+    Created At:         2024-01-30T12:05:14Z
+    Port:               50051
+    Protocol:           grpc
+    Service Name:       operatorhubio-catalog
+    Service Namespace:  openshift-operators
+Events:                 <none>
+----
+
+. Confirm that the catalog source is running by using the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-operators -l olm.catalogSource=operatorhubio-catalog
+----
++
+.Example output
+[source,terminal]
+----
+NAME                          READY   STATUS    RESTARTS   AGE
+operatorhubio-catalog-5tswm   1/1     Running   0          99m
+----
+
+. Create a Subscription CR configuration file by using the following example YAML:
++
+.Example Subscription custom resource YAML
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: my-cert-manager
+  namespace: olm-microshift <1>
+spec:
+  channel: stable
+  name: cert-manager
```

Comment:
@ShaunaDiaz please do not use cert-manager as example because it only support all namesapce mode.
here is my checking which is from csv of operator
```console
      displayName: cert-manager
      installModes:
      - supported: false
        type: OwnNamespace
      - supported: false
        type: SingleNamespace
      - supported: false
        type: MultiNamespace
      - supported: true
        type: AllNamespaces
```
while the example procedure here is for own namesapce mode because you said
1, The Operator uses content from the global namespace, but runs only in the `olm-microshift` namespace
2, you define og as 
```yaml
kind: OperatorGroup
apiVersion: operators.coreos.com/v1
metadata:
  name: og
  namespace: olm-microshift
spec: <1>
  targetNamespaces:
  - olm-microshift
```
so the installation fails.


one general suggestion: please execute the step before you work out to see if it works or not. it apply for section `Adding OLM-based Operators to a networked cluster in a specific namespace` and `Adding OLM-based Operators to a networked cluster using the global namespace`. or else, the step and output are possible wrong.

Reply:
@kuiwang02 Apologies, I do not have a working test environment or VM that runs MicroShift.

Reply:
@ShaunaDiaz never mind :)
you could use https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/microshift/job/microshift-deployment-pipeline to create env.
here is the build https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/microshift/job/microshift-deployment-pipeline/4396/ I used to create env for example

this jenkins job is maintained by @jogeo and @dhensel-rh (thanks). I think you could contact with them if there is problem.